README.md

```diff
@@ -44,5 +44,11 @@ currently specified in the `.gitmodules` file.
 For detailed walkthroughs of git submodule usage, see the
 [Git Tools documentation](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
 
+## Building the Library
+Use the provided scripts `build-release.sh` and `build-debug.sh`. Ensure that `cmake` is installed first.
```

Comment:
I suggest rephrasing this to:
* Put the "first" step at the beginning 
* Clarify how to choose which build script a user should run

```suggestion
Ensure that `cmake` is installed. To create an optimized build without debug symbols, run the `build-release.sh` script. To create an unoptimized build that includes debug symbols, use `build-debug.sh`.
```

Is there a minimum version of `cmake`? Should we have a `dependencies` section that spells out platform and compiler versions? (Not necessarily authored in this PR, we can open an issue to track it.)

Reply:
I don't know.

Reply:
I'd be happy with a "known to work on" section.